starapi/viewsets.py

```diff
@@ -88,6 +93,35 @@ class SampleAnnotationViewSet(viewsets.ViewSet):
         'annotation': 'annotation',
     }
 
+    @transaction.atomic
+    def create(self, request):
+        user_id = request.user.id
+
+        serializer = SampleAnnotationValidator(
+            data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        series = serializer.validated_data['series']
+        tag = serializer.validated_data['tag']
+
+        series_tag = first(SeriesTag.objects.filter(series=series, tag=tag))
```

Comment:
Also MAJOR error not filtering by platform here. SeriesTag is defined by these 3 things: serie, platform, tag. And filter by `is_active=True`.